onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -28,6 +28,7 @@
 #include "core/optimizer/shape_to_initializer.h"
 #include "core/optimizer/slice_elimination.h"
 #include "core/optimizer/unsqueeze_elimination.h"
+#include "core/optimizer/reshape_fusion.h"
 #include "core/platform/env.h"
```

Comment:
Is there meant to be a unit test added here?

Reply:
Yes. Previous merge with master has problem, which removed unit test. I'll add it back.